serialfile.go

```diff
@@ -122,7 +132,7 @@ func (f *serialFile) NextFile() (string, Node, error) {
 	stat := f.files[0]
 	f.files = f.files[1:]
 
-	for !f.handleHiddenFiles && strings.HasPrefix(stat.Name(), ".") {
+	for !f.filter.IncludeHidden && strings.HasPrefix(stat.Name(), ".") {
```

Comment:
I realize this is from existing code, but I think it may suffer from the same issue as above - it doesn't take account of handling hidden files in Windows. It would be best to consolidate this check in one place that handles all OSes

Reply:
I have yet to test on windows, but I refactored `Filter.ShouldExclude` to use the existing `isHidden` function, which seems to have a Windows variant that will be used in Windows builds.